ibm_was/README.md

```diff
@@ -32,11 +34,17 @@ Once you've made this change, click "Apply" to save the configuration and restar
 
 ### Configuration
 
-1. Edit the `ibm_was.d/conf.yaml` file, in the `conf.d/` folder at the root of your Agent's configuration directory to collect your IBM WAS performance data. See the [sample ibm_was.d/conf.yaml][2] for all available configuration options.
+Follow the instructions below to configure this check for an Agent running on a host. For containerized environments, see the [Containerized](#containerized) section.
```

Comment:
See this Trello ticket for more context for my comment: https://trello.com/c/4DGLu3Ql/1913-show-examples-of-using-the-initconfig-parameter-in-autodiscovery

We change this Configuration paragraph for all AD-enabled integrations to point the users to the Containerized section. However, in none of the docs we actually show how to use the conf.yaml in the AD integration template (which is what this paragraph is talking about). We should take care of this asap IMHO, otherwise the current flow is broken and confusing.

@clamoriniere can you help us with an example of how to use init_config with conf.yaml? I had a chat with Cecilia a week ago and she mentioned that she asked @CharlyF about it but they couldn't find an answer yet.

Reply:
This will be resolved in a different PR.